server/src/controllers/__tests__/user.spec.ts

```diff
@@ -104,8 +104,10 @@ describe("UserController", () => {
   })
 
   it("should get a user", async () => {
-    const mockUser = { id: 1, email: "test@test.com" }
-    const mockScore = { userId: 1, score: 0 }
+    const date = new Date()
+    date.setUTCHours(0, 0, 0, 0)
+    const mockUser = { id: 1, email: "test@test.com", streak: 1 }
+    const mockScore = { userId: 1, score: 2, day: date }
```

Comment:
We expect the mock score to include an `id` field. I'm only mentioning this because our model design has a PK, which allows it to be unique. **In relational databases, you cannot have duplicate PK's**, so if a `userId` had multiple scores, we need that PK to be unique.

![image](https://github.com/zdigness/puttdle/assets/23369330/33ce2f85-56ed-46e8-82a4-1e5c6cccc5ab)